azure-pipelines.yml

```diff
@@ -5,20 +5,24 @@ jobs:
 - template: .azure/default-build.yml
   parameters:
     agentOs: Windows
+    cMakeRunArgs: '-A x64'
     beforeBuild:
-    - script: echo Hello, world!
-      displayName: 'Run a one-line script'
+    - powershell: "& vcpkg.exe install cpprestsdk cpprestsdk:x64-windows --vcpkg-root ./submodules/vcpkg"
+      displayName: vcpkg install dependencies
 
 - template: .azure/default-build.yml
   parameters:
     agentOs: macOs
     beforeBuild:
-    - script: echo Hello, world!
-      displayName: 'Run a one-line script'
+    - script: brew install gcc boost
+    - bash: "./submodules/vcpkg/bootstrap-vcpkg.sh"
+      displayName: Bootstrap vcpkg
+    - bash: "./submodules/vcpkg/vcpkg install cpprestsdk"
```

Comment:
Note that this will still use boost from inside vcpkg -- you may want to avoid installing boost from brew above to minimize the chance of contamination.

Reply:
Does the same apply for Linux?

Reply:
Yes, though afaict you are already installing boost via vcpkg (and not apt) on Linux.

Reply:
Oh right, I forgot my own PR, we install boost with vcpkg.

I'll remove boost from brew